keras_nlp/utils/text_generation_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Text Generation Utils."""
+
+
+import tensorflow as tf
+
+from keras_nlp.utils.text_generation import generate_text_greedy
+
+
+class TextGenerationTest(tf.test.TestCase):
```

Comment:
interesting, do you need tf.test.main()?

Reply:
Seems no - our test is based on pytest, which automatically captures the test cases.